west.yml

```diff
@@ -23,6 +23,8 @@ manifest:
       url-base: https://github.com/zephyrproject-rtos
     - name: babblesim
       url-base: https://github.com/BabbleSim
+    - name: nandojve
+      url-base: https://github.com/nandojve
```

Comment:
This is probably an omission? Or maybe pending that the hal gets submitted upstream?

Reply:
It is not an omission, it is part of pending hal submission. Once hal is in place this will be replaced as last part of this pull request. The hal is already checked using the [license_check](https://github.com/nandojve/hal_bouffalolab/actions/workflows/license_check.yml) workflow and it is green which helps to speed up the process.

Reply:
What would be the next step for contributing the HAL? Is this something discussed i.e. on Discord or Mailing Lists?

Reply:
When this is ready someone from the project will create the https://github.com/zephyrproject-rtos/hal_bouffalolab. I'll initialize the repository with the content of https://github.com/nandojve/hal_bouffalolab. Then I update this PR with the normal reference and hash to get this merge.

Once it in place, it is normal development flow.

However, if you already wants to start to work you can. Just clone my reference and then later add the zephyr remote. It is not big deal.